ci/scripts/extract_version.sh

```diff
@@ -15,6 +18,13 @@ fi
 
 # Extract major and minor version
 major_minor=$(echo "$version" | cut -d. -f1,2)
+full_version=$(echo "$version" | cut -d. -f1,2,3)
+
+if [[ "$version_type" == "major_minor" ]]; then
+    semver_version="$major_minor"
+else
+    semver_version="$full_version"
+fi
```

Comment:
```suggestion
full_version=$(echo "$version" | cut -d. -f1,2,3)

if [[ "$version_type" == "major_minor" ]]; then
    echo $major_minor
else
    echo $full_version
fi
```

Then delete the echo `echo "$semver_version"` below.

No need to store it in a variable right? Also the variable name is misleading since it's possibly not a semver, it could be just a major_minor.